nmostesting/suites/IS1201Test.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (C) 2023 Advanced Media Workflow Association
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from time import sleep
+
+from .. import Config as CONFIG
+from ..GenericTest import GenericTest
+from ..IS04Utils import IS04Utils
+from ..TestHelper import WebsocketWorker
+
+NODE_API_KEY = "node"
+CONTROL_API_KEY = "control"
+
+
+class IS1201Test(GenericTest):
+
+    def __init__(self, apis, **kwargs):
+        # Remove the RAML key to prevent this test suite from auto-testing IS-04 API
+        apis[NODE_API_KEY].pop("raml", None)
```

Comment:
Interesting idea... should we be doing this in all the "interaction" test suites? @AMWA-TV/nmos-architecture-review Discuss!